homeassistant/components/switchbot/sensor.py

```diff
@@ -97,9 +103,8 @@ def __init__(
         super().__init__(coordinator)
         self._sensor = sensor
         self._attr_unique_id = f"{coordinator.base_unique_id}-{sensor}"
-        name = coordinator.device_name
-        self._attr_name = f"{name} {sensor.replace('_', ' ').title()}"
         self.entity_description = SENSOR_TYPES[sensor]
+        self._attr_name = self.entity_description.name
```

Comment:
This is the default behavior of entities already, so there is no need to manually set it.

```suggestion
```